connect/api/v1/internal/integrations/integrations_rest_client.py

```diff
@@ -0,0 +1,47 @@
+from django.conf import settings
+
+import requests
+
+from connect.api.v1.internal.internal_authentication import InternalAuthentication
+
+
+class IntegrationsRESTClient:
+
+    def __init__(self):
+        self.base_url = settings.INTEGRATIONS_REST_ENDPOINT
+        self.authentication_instance = InternalAuthentication()
```

Comment:
This violates one of the principles of S.O.L.I.D. the Open-Closed Principle. please add this to the backlog.
Example: https://betterprogramming.pub/the-open-closed-principle-explained-in-python-f5517488f990